crates/cli/src/model/cli_configuration.rs

```diff
@@ -150,6 +150,7 @@ mod tests {
                 tree_sitter_query_base64: None,
                 arguments: vec![],
                 tests: vec![],
+                is_testing: Some(false),
```

Comment:
From the API, it will always be true or false cuz we're not using omitempty or a bool pointer.